### DISCUSSION TITLE
notes column in list view

### DISCUSSION THREAD

**Discussion by zhawzid-ottj** [ComfortablyCoding/strapi-plugin-notes#5] [Ideas]
It would be nice if we could add the notes as a column to the list of records.

**Comment by ComfortablyCoding** [comment]
Unfortunately that is not possible at this time. Notes are not a custom field (it was done before they were released) but a plugin utilizing injection zones. 

I don't believe their is any easy way to migrate to use custom fields instead but I will look into it.
